radiation/TwoStreamMLPEMod.F90

```diff
@@ -370,9 +370,13 @@ function GetRdDn(this,ican,icol,ib,vai) result(r_diff_dn)
            scelb%B1*scelb%lambda2_diff*exp(-scelb%a*vai))
 
       if(debug)then
+
          ! if(isnan(r_diff_dn))then  !RGK: NVHPC HAS A BUG IN THIS INTRINSIC (01-2024)
+         ! if(shr_infnan_isnan(r_diff_dn)) then !RGK: this statement simply didn't work
+         ! Dont trigger an endrun, we flag an error with r_diff_dn,
```

Comment:
@rgknox I'm looking at this code and just noticing that the comment here says not to trigger an endrun, but it still seems like it does end up with an endrun anyway?

Reply:
Will update comments in future pr since tests are complete